spec/controllers/my/shares_controller_spec.rb

```diff
@@ -2,29 +2,48 @@
 
 describe My::SharesController, :type => :controller do
   describe "logged in user" do
+    let(:user) { FactoryGirl.find_or_create(:archivist) }
+
     before (:each) do
-      @user = FactoryGirl.find_or_create(:archivist)
-      sign_in @user
+      sign_in user
     end
 
     describe "#index" do
       before do
         GenericFile.destroy_all
         Collection.destroy_all
-        @other_user = FactoryGirl.create(:user)
-        @my_file = FactoryGirl.create(:generic_file, depositor: @user)
-        @unshared_file = FactoryGirl.create(:generic_file, depositor: @other_user)
-        @shared_with_me = FactoryGirl.create(:generic_file).tap do |r|
-          r.apply_depositor_metadata @other_user
-          r.edit_users += [@user.user_key]
+      end
+
+      after do
+        GenericFile.destroy_all
```

Comment:
These are unnecessary on the fedora-4 branch, and cause the test suite to run slower. We use ActiveFedora::Cleaner instead. Please remove them.